source/_integrations/sms.markdown

```diff
@@ -26,6 +26,36 @@ This integration provides the following platforms:
 
 ## Notifications
 
+An SMS message can be send by calling the `notify.sms`. It will send the message to all phone numbers specified in the `target` parameter.
```

Comment:
```suggestion
An SMS message can be sent by calling the `notify.sms`. It will send the message to all phone numbers specified in the `target` parameter.
```